tests/MyGame/Example/Monster.go

```diff
@@ -231,6 +265,14 @@ func (rcv *Monster) Testbool() byte {
 	return 0
 }
 
+func (rcv *Monster) MutateTestbool(n byte) bool {
+	o := flatbuffers.UOffsetT(rcv._tab.Offset(34))
+	if o != 0 {
+		 return rcv._tab.MutateByte(o + rcv._tab.Pos, n)
```

Comment:
You could roll the looking up of the offset and the if-then into MutateByte, rather than it being in generated code.